### ISSUE TITLE
[Request] Integration with easy-llama

### ISSUE THREAD

**Issue by iwr-redmond** [noamgat/lm-format-enforcer#165]
The [easy-llama](https://github.com/ddh0/easy-llama) project is an alternative to llama-cpp-python, which has been poorly maintained in recent times.

The maintainer has [added support for logits](https://github.com/ddh0/easy-llama/issues/10#issuecomment-2623558490), and it should be possible to develop an integration for LM Format Enforcer as a result.
